tests/test_parse_osm.py

```
@@ -400,21 +449,18 @@ async def test_parse_miscellaneous_sets_attrs(osm_parser, expected_attr, expecte
-            lambda k: "UniqueName" if k == ATTR_PLACE_NAME else None,
-            True,
-        ),
-        ("duplicate", "DupeName", lambda k: False, lambda k: "DupeName", False),
+        ("unique", "UniqueName", True),
+        ("duplicate", "DupeName", False),
     ],
 )
 async def test_set_place_name_no_dupe_param(
-    osm_parser, case, current_name, is_blank, existing_get_attr, should_set
-):
+    osm_parser: OSMParserFactory,
```

Ruff (0.15.12)

[warning] Unused function argument: `osm_parser`

(ARG001)